### PR TITLE
Update changelog after 1.5.7 release

### PR DESCRIPTION
**Pull request by awaelchli**
## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
I made sure I had fun coding 🙃

Part of #1 (it's a lie, this is just here to avoid noisy GitHub bot)

cc @borda

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -320,40 +320,32 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
-- Fixed an `AttributeError` occuring when using a `CombinedLoader` (multiple dataloaders) for prediction ([#11111](https://github.com/PyTorchLightning/pytorch-lightning/pull/11111))
-
-
-- Fixed bug where `Trainer(track_grad_norm=..., logger=False)' would fail ([#11114](https://github.com/PyTorchLightning/pytorch-lightning/pull/11114))
-
-
 - Fixed logging on `{test,validation}_epoch_end` with multiple dataloaders ([#11132](https://github.com/PyTorchLightning/pytorch-lightning/pull/11132))
 
 
 - Fixed double evaluation bug with fault-tolerance enabled where the second call was completely skipped ([#11119](https://github.com/PyTorchLightning/pytorch-lightning/pull/11119))
```
Comment (rohitgr7) [inline CHANGELOG.md:329]: these were missed in last release?

**Review by rohitgr7 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (awaelchli) [reply CHANGELOG.md:329]: https://github.com/PyTorchLightning/pytorch-lightning/pull/11132 doesn't have the 1.5.x and https://github.com/PyTorchLightning/pytorch-lightning/pull/11119 couldn't be included due to conflicts (I will change milestone)

**Review by carmocca — APPROVED** [review]

**Review by ananthsub — APPROVED** [review]

**Review by Borda — APPROVED** [review]
